### PR TITLE
pull request template: consolidate Context & Summary of Changes

### PR DESCRIPTION
**Pull request by nathanshelly**
## Context

> Why are you making this change?

To simplify our template and remove an often redundant section.

## Origin

> What team/individual requested this change? Please link to a JIRA ticket or other documentation if applicable. If you identified this issue yourself please write "Me" (and still link to any additional context if it exists).

This will be deleted

## Summary of Changes

This will also be deleted

## Test Plan

N/A

## Checklist

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to external documentation if necessary (READMEs, Confluence, etc.)
- [x] I have checked that other PRs this PR depends on have already been deployed.

### CODE REVIEW

**Review by findzaatar — APPROVED** [review]

**Review by sidbai — APPROVED** [review]

**Comment by xinxingjiang1** [comment]
This is great!

Can we merge origin too? It is almost always "Me". And with the new requirement for a Jira ticket, it is pretty clear the Jira creator is the origin.

**Review by xinxingjiang1 — APPROVED** [review]

**Review by markrmullan — APPROVED** [review]
LGTM— code Danger (and maybe web Danger too?) will need to be updated based on these changes https://github.com/opendoor-labs/code/blob/f085235f93f91fb42dccec4c4a66933657ff47f8/code/pipeline/danger/Dangerfile#L106-L129

**Comment by findzaatar** [comment]
Danger for sox compliance, is looking for origin and context specifically

**Comment by nathanshelly** [comment]
good point y'all, opened PRs to update both the danger checks

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-/PULL_REQUEST_TEMPLATE.md @opendoor-labs/cloud-infra
+/PULL_REQUEST_TEMPLATE.md @opendoor-labs/foundation
```
Comment (markrmullan) [inline CODEOWNERS:1]: nice

**Pull request merged**